config/rbac/role.yaml

```diff
@@ -4,6 +4,18 @@ kind: ClusterRole
 metadata:
   name: manager-role
 rules:
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
```

Comment:
Presumably this is only ever needed for testing? Unsure if this should be added into what will eventually be a real deployments rbac.

Reply:
only for testing yes. I could not figure out the way to grant those permissions only when we decide to override the transport layer so I went with the second best :thinking: Would you have an idea how to improve? My thoughts were to use a secret with string data but add rbacs seemed like a better solution. WDYT?